src/common/classes/fb_string.h

```diff
@@ -411,10 +411,10 @@ namespace Firebird
 		}
 		void insert(size_type p0, const_pointer s, const size_type n)
 		{
```

Comment:
IMO this fix should be done as separate pull request and backported to 2.5 and 3.0.